Disable the restart functionality so the daemon can manage it.

---
 src/NzbDrone.Core/Lifecycle/LifecycleService.cs | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/NzbDrone.Core/Lifecycle/LifecycleService.cs b/src/NzbDrone.Core/Lifecycle/LifecycleService.cs
index 9ed36a42e..565641284 100644
--- a/src/NzbDrone.Core/Lifecycle/LifecycleService.cs
+++ b/src/NzbDrone.Core/Lifecycle/LifecycleService.cs
@@ -46,8 +46,7 @@ public void Shutdown()
         public void Restart()
         {
             _logger.Info("Restart requested.");
-
-            _eventAggregator.PublishEvent(new ApplicationShutdownRequested(true));
+            _eventAggregator.PublishEvent(new ApplicationShutdownRequested());

             if (_runtimeInfo.IsWindowsService)
             {
